cmd/worker.go

```diff
@@ -9,14 +9,15 @@ import (
 	"github.com/tsaikd/gogstash/config/goglog"
 )
 
-func waitSignals(ctx context.Context) error {
+func waitSignals(ctx context.Context) (os.Signal, error) {
```

Comment:
returned error is useless?

Reply:
pre-commit would have told me to remove it 😇